cmd/config_helper.go

```diff
@@ -50,11 +50,21 @@ func toConfig(install, forceIgnored []*api.Package, targets []string, cmdline []
 	}
 
 	providers := collectProviders(forceIgnored, install)
-	packageNames := sortedKeys(allPackages)
-	sortedPackages := make([]*bazeldnf.RPM, 0, len(packageNames))
-	for _, name := range packageNames {
+	slices.Sort(targets)
+	sortedPackages := make([]*bazeldnf.RPM, 0, len(allPackages))
```

Comment:
maybe move this down to where sortedPackages is constructed so it's a bit more clear what's happening